src/arrayMethodIncludes.js

```diff
@@ -5,8 +5,21 @@
  */
 function applyCustomIncludes() {
   [].__proto__.includes2 = function(valueToFind, fromIndex) {
```

Comment:
```suggestion
  [].__proto__.includes2 = function(valueToFind, fromIndex = 0) {
```

установит 0 по умолчанию, если фромИндекс не был передан